src/app/core/services/youtube.search.spec.ts

```diff
@@ -2,13 +2,20 @@
  * Testing a Service
  * More info: https://angular.io/docs/ts/latest/guide/testing.html
  */
-import { TestBed, inject } from '@angular/core/testing';
-import { HttpClientModule } from '@angular/common/http';
+import { 
```

Comment:
Consider keeping the import statements on a single line for better readability, unless the line length exceeds the recommended limit (usually 80-120 characters).